src/main/java/io/ddavison/conductor/util/DriverUtil.java

```diff
@@ -62,7 +67,7 @@ public static WebDriver getDriver(ConductorConfig config) {
                     break;
                 case EDGE:
                     EdgeOptions edgeOptions = new EdgeOptions();
-                    setCustomCapabilities(config, edgeOptions);
+                    setCustomCapabilities(config, edgeOptions, desiredCapabilities);
```

Comment:
You are missing the `capabilities = `, combine this and the next line, like we did on the `SAFARI` case.